src/api/checkout.js

```diff
@@ -7,13 +7,13 @@ app.post('/', (req, res) => {
     if (typeof req.body.studentId !== 'number') {
         return res.status(400).failureJson('A student id is required.');
     }
-    if (typeof req.body.itemAddresses !== 'object'
-        || typeof req.body.itemAddresses[0] !== 'string') {
+    if (typeof req.body.equipmentAddresses !== 'object'
+        || typeof req.body.equipmentAddresses[0] !== 'string') {
```

Comment:
How should we check out various quantities of a model? Do we need to include multiple copies of the model address in `equipmentAddresses`? Having an address / quantity pair would be more manageable.